test/tests_basic.jl

```diff
@@ -74,6 +102,84 @@ end
     @test_nowarn p4est_destroy(p4est)
     @test_nowarn p4est_connectivity_destroy(connectivity)
   end
+
+  @testset "p8est_connectivity_new_unitcube" begin
+    connectivity = @test_nowarn p8est_connectivity_new_unitcube()
+    @test_nowarn p8est_connectivity_destroy(connectivity)
+  end
+
+  @testset "p4est_refine_coarsen" begin
```

Comment:
```suggestion
  @testset "p4est_refine and p4est_coarsen" begin
```
We do not test a function `p4est_refine_coarsen` here (does something like that exist?) so the current name of the testset may be a bit confusing.